src/gluonts/itertools.py

```diff
@@ -128,6 +129,23 @@ def get_batch():
     return iter(get_batch, [])
 
 
+@dataclass
+class Chained:
```

Comment:
I think it's nicer to use the same name as in `itertools`.

```suggestion
class Chain:
```

Reply:
@jaheba applied the change, although I had chosen `Chained` for grammatical consistency with `Cached`...

Reply:
Maybe `Cache` would also be better? `Cylic` should probably also be `Cycle`, but we have to start somewhere!

Also, I like `Chain` more than `Chained` regardless :).